apps/prairielearn/src/lib/editors.ts

```diff
@@ -1111,7 +1123,11 @@ export class QuestionRenameEditor extends Editor {
     assert(this.question.qid, 'question.qid is required');
 
     debug('QuestionRenameEditor: write()');
-    const questionsPath = path.join(this.course.path, 'questions');
+
+    let questionsPath = path.join(this.course.path, 'questions');
+    if (this.question.is_draft) {
+      questionsPath = path.join(questionsPath, '__drafts__');
+    }
```

Comment:
As with the other editor here, I don't think this needs to be aware of whether or not something is a draft. The QID will simply be `__drafts__/draft_...`.

Reply:
Switched systems in a94d9c043b56e640362369ff73f52d0a019803dd.